api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -22,29 +25,40 @@
  * @since 1.5
  * @see OpenmrsMetadata
  */
+@MappedSuperclass
 public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements OpenmrsMetadata {
 	
 	//***** Properties *****
+	@Column(name = "name", nullable = false, length = 255)
 	@Field
 	private String name;
 	
+	@Column(name = "description", length = 255)
```

Comment:
@rkorytkowski
@wluyima
I created an issue for overriding the description value in concrete classes
https://issues.openmrs.org/browse/TRUNK-4674